src/analysis/normed/group/basic.lean

```diff
@@ -1516,6 +1516,14 @@ by simp only [←dist_one_right, dist_pi_le_iff hr, pi.one_apply]
 @[to_additive pi_nnnorm_le_iff]
 lemma pi_nnnorm_le_iff' {r : ℝ≥0} : ∥x∥₊ ≤ r ↔ ∀ i, ∥x i∥₊ ≤ r := pi_norm_le_iff' r.coe_nonneg
 
+@[to_additive pi_norm_le_iff''] lemma pi_norm_le_iff''' [nonempty ι] : ∥f∥ ≤ r ↔ ∀ b, ∥f b∥ ≤ r :=
+begin
+  by_cases hr : 0 ≤ r,
+  { exact pi_norm_le_iff' hr },
+  { exact iff_of_false (λ h, hr $ (norm_nonneg' _).trans h)
+      (λ h, hr $ (norm_nonneg' _).trans $ h $ classical.arbitrary _) }
+end
```

Comment:
Might be nice to have the `nnnorm` version of this as well, for convenient `rw`

Reply:
Not sure what you mean. `pi_nnnorm_le_iff` already exists, it's just above.